app/views/shared/_header.html.erb

```diff
@@ -41,7 +41,7 @@
           </li>
         <% end %>
         <li class="<%= 'active' if current_page?(controller: '/deploys', action: 'search') %>">
-          <%= link_to "Deploys", search_deploys_path %>
+          <%= link_to "Deploying", search_deploys_path %>
```

Comment:
I was suggesting `Deploys` for the search and `Deploying` for the active deploys